test/python/expected/sql_validation_framework/expected_create.out

```diff
@@ -73,6 +73,7 @@ Could not find tests for table sys.babelfish_syslanguages
 Could not find tests for table sys.service_settings
 Could not find tests for table sys.spt_datatype_info_table
 Could not find tests for table sys.versions
+Could not find upgrade tests for function typeproperty
```

Comment:
We don't need this entry here, instead add the upgrade test for typeproperty. Add the test name in the required schedule files.

Reply:
Added the upgrade test.